### DISCUSSION TITLE
Vivado v2015.2 Version?

### DISCUSSION THREAD

**Discussion by AjayShanker-geek** [NUS-CG3207/labs#2] [Wrangling Vivado] [answered]
Hi, the materials did not state which specific Vivado version we should install or Ubuntu version to use. Only through the repo with `vivado.log` but its back from `S1_AY2016-17`.

**Comment by NeilBaner** [comment, marked as answer]
See https://github.com/NUS-CG3207/lab-skeletons/discussions/1. Realized it's not really visible once I close it, so I'll reopen it for others' benefit.

**Comment by AjayShanker-geek** [comment]
Thanks, I will this discussion.
